cli/command/task/print.go

```diff
@@ -10,33 +10,40 @@ import (
 	"github.com/docker/cli/cli/command/idresolver"
 	"github.com/docker/cli/cli/config/configfile"
 	"github.com/docker/docker/api/types/swarm"
+	"vbom.ml/util/sortorder"
 )
 
-type tasksBySlot []swarm.Task
+type tasksSortable []swarm.Task
 
-func (t tasksBySlot) Len() int {
+func (t tasksSortable) Len() int {
 	return len(t)
 }
 
-func (t tasksBySlot) Swap(i, j int) {
+func (t tasksSortable) Swap(i, j int) {
 	t[i], t[j] = t[j], t[i]
 }
 
-func (t tasksBySlot) Less(i, j int) bool {
-	// Sort by slot.
-	if t[i].Slot != t[j].Slot {
-		return t[i].Slot < t[j].Slot
+func (t tasksSortable) Less(i, j int) bool {
+	if t[i].Name != t[j].Name {
+		return sortorder.NaturalLess(t[i].Name, t[j].Name)
```

Comment:
Hello @SaretX, thank you for your contribution 👍 
I think that as we update the sort behavior here, at one test should have been updated or added to reflect that change 🤔 
Can you update your PR with that test? Please tell me if you need help for that 👍